gui-js/libs/core/src/lib/services/communication/communication.service.ts

```diff
@@ -470,10 +470,8 @@ export class CommunicationService {
     this.awaitingZoom = false;
 
     // schedule resetScroll when zooming stops
-    if (!this.resetScrollWhenIdle)
-      this.resetScrollWhenIdle = setTimeout(() => { var self = this; self.resetScroll(); self.resetScrollWhenIdle = null; }, 100);
-    else
-      this.resetScrollWhenIdle.refresh();
+    if (this.resetScrollWhenIdle) clearTimeout(this.resetScrollWhenIdle);
+    this.resetScrollWhenIdle = setTimeout(() => { var self = this; self.resetScroll(); self.resetScrollWhenIdle = null; }, 100);
```

Comment:
Been a while since I made this change, but I think I recall your approach simply didn't work in some situations... I will rewrite my code to re-use a singularly defined method, though.